### ISSUE TITLE
After adding a jar, the implicit kernel val is not getting picked up

### ISSUE THREAD

**Issue by Lull3rSkat3r** [ibm-et/spark-kernel#68]
When there is a function taking the kernel as an implicit value

```scala
def myfunc(implicit sparkKernel: SparkKernel){
// function body
}
```

When I run `%%AddJar some.jar` my function says it cannot find the implicit value.

**Comment by rcsenkbeil** [comment]
Should be a quick check of `isImplicit` from the symbol that we can pull out via `interpreter.symbolOfTerm("sparkKernel")`. If it's true, then we add the `implicit` modifier.

**Comment by rcsenkbeil** [comment]
Resolved via commit 4785c4d713307b6412a3934094ed0fb8f8a6124c.
